apps/ledger-live-mobile/src/helpers/getDeviceAnimation.ts

```diff
@@ -0,0 +1,52 @@
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
```

Comment:
I created it because `getDeviceAnimation`for the DeviceAction is located inside `apps/ledger-live-mobile/src/components/DeviceAction/` which created 2 problems:
- I did not want to add stuff not related to DeviceAction inside `apps/ledger-live-mobile/src/components/DeviceAction/`
- I did not want to move the DeviceAction `getDeviceAnimation` method outside of `apps/ledger-live-mobile/src/components/DeviceAction/` because it was going to update a lot of files using this function

The second reason is maybe less problematic. If you think we should not have 2 getDeviceAnimation, then i'll move it inside `apps/ledger-live-mobile/src/helpers/` ? and merge it with the other one. I'm not against this. WDYT ?

Though, a last argument against merging the 2 is: the animation for the DeviceAction requires to exist for all the existing devices. The animation for none-DeviceAction does not require this, and can exist only on certain devices. IMO it's easier to handle into 2 `getDeviceAnimation` because of this

Reply:
> I did not want to add stuff not related to DeviceAction inside apps/ledger-live-mobile/src/components/DeviceAction/
With this one I agree.
> I did not want to move the DeviceAction getDeviceAnimation method outside of apps/ledger-live-mobile/src/components/DeviceAction/ because it was going to update a lot of files using this function
Is it really that many? I'd think all the files that eventually use this function currently would be insideapps/ledger-live-mobile/src/components/DeviceAction/ ... If that's not the case I'd say that this function should already have been extracted from this folder long ago, as it doesn't serve only the "DeviceAction" component anymore. In any case, better late than never, I'm not sure that "this would change many imports" should prevent us from doing it.

>The second reason is maybe less problematic. If you think we should not have 2 getDeviceAnimation, then i'll move it inside apps/ledger-live-mobile/src/helpers/ ? and merge it with the other one. I'm not against this. WDYT?
Yep, that's what I think we should do.

For the last argument, still don't think it's enough for us not to merge them.

Reply:
Yes now there would be other files outside `apps/ledger-[live-mobile](https://ledgerhq.atlassian.net/browse/LIVE-mobile)/src/components/DeviceAction/` 

I agree, i'm going to try to extract it (you good with me putting into `apps/ledger-live-mobile/src/helpers/` ?) and let you know when it's done ;)